styles/globals.css

```diff
@@ -922,4 +926,71 @@ body.dark-theme .ant-menu .ant-menu-submenu-arrow {
 
 .tox-statusbar__text-container {
   display: none !important;
+}
+.antSteps .ant-steps .ant-steps-item-wait .ant-steps-item-container .ant-steps-item-icon .ant-steps-icon {
```

Comment:
these are global changes, please make sure design doesn't get messed at other places in the site